### DISCUSSION TITLE
Problems with sntp

### DISCUSSION THREAD

**Discussion by viniciusfl** [JAndrassy/WiFiEspAT#90] [Q&A]
I've been trying to use the getTime() function but it keeps forever in the "while(WiFi.getTime() < SECS_YR_2000") loop.
I think that i've connected to the internet correctly. I relied on the file SNTPTime.ino but i'm not using SoftwareSerial to communicate the Arduino with ESP. I'm using an Arduino Uno WiFi R3 that can use the board serial to communicate between the ATmega and ESP. 
I tried the firmware checker and its said that my firmware was fine (Espessif Nonos latest version), so i don't know why the function getTime isn't able to get the time. 
Could you help me?

**Comment by JAndrassy** [comment]
getTime doesn't work with standard AT 1.7, because the AT firmware doesn't return 'epoch' time value.
